scripts/legislation/africa/sierra_leone_english.sh

```diff
@@ -0,0 +1,527 @@
+#!/bin/bash
+
+function remove_all_text_before_first_header {
+  sed -n '/^Interprelation./,$p' | \
```

Comment:
Did you intend for the period to match a period specifically or any character? If the former, then you need to escape the period (i.e. `\.`).

Reply:
Thank you for pointing this out. I have deleted the period.